ci/rpm/scan_daos_node.sh

```diff
@@ -14,6 +14,18 @@ pushd "/var/tmp/${lmd_src}"
 sudo ./install.sh
   sudo ln -s /usr/local/maldetect/ /bin/maldet
 popd
+
+fc_conf="/etc/freshclam.conf"
+sudo chmod 766 "$fc_conf"
+if ! grep -q 'ScriptedUpdates no' "$fc_conf"; then
+  printf "ScriptedUpdates no\n" >> "$fc_conf"
+fi
```

Comment:
```suggestion
if ! grep -q 'ScriptedUpdates no' "$fc_conf"; then
  sudo bash -c "echo \"ScriptedUpdates no\" >> \"$fc_conf\""
fi
```

`printf` is overkill for just adding a line with no formatting.